jaggery-modules/rxt/module/scripts/core/core.js

```diff
@@ -991,6 +991,32 @@ var core = {};
         }
         return false;
     };
+    /**
+     * Returns an Object with a mapping of sortable field name and the corresponding label name.This mapping contains
+     * the fields need to be displayed in the sort dropdown menu in the asset list page.
+     * If the field is not found then an empty array is returned
+     * @example
+     *     var value = rxtManager.getSortingNames('gadget');
+     * @param  {String} type The RXT type
+     * @return {Object}  The list of sortable fields with their associated label names, available for the give RXT type
+     */
+    RxtManager.prototype.getSortingAttributes = function(type) {
+        var rxtDefinition = this.rxtMap[type];
+        if (!rxtDefinition) {
+            log.error('Unable to locate the rxt definition for type: ' + type +
+                ' in order to return sortable field names');
+            throw 'Unable to locate the rxt definition for type: ' + type + ' in order to return sortable field names ';
```

Comment:
Thank you for pointing out this issue, I have fixed it with [this commit](https://github.com/wso2/carbon-store/pull/842/commits/20e3ef91eec7d92300a340261337afb6ac9a8f34)